packages/chaire-lib-backend/src/models/db/default.db.queries.ts

```diff
@@ -207,7 +207,7 @@ const updateMultiple = async <T extends GenericAttributes, U>(
 
 const deleteRecord = async (knex: Knex, tableName: string, id: string) => {
     try {
-        if (!uuidValidate(id)) {
+        if (!uuidValidate(id) && (await knex(tableName).where('id', id)).length < 1) {
```

Comment:
Not sure why the second part is added. Can we add a comment in the code ?

Reply:
Removed this part since deletion is now handled directly in tokens